wallmatrix/sources/mbta.py

```diff
@@ -7,31 +8,41 @@
 from wallmatrix.sources import Source
 
 
+class Line(TypedDict):
+    color: str
+    line: str
+    route: str
+    origin: str
+    destination: str
+    direction: int
+    walk_min: int
+
+
 class MBTA(Source):
     SOURCE_NAME = "MBTA North from NEU"
 
-    LINES = [
+    LINES: list[Line] = [
         {
             "color": "ED8B00",
             "line": "line-Orange",
```

Comment:
We could replace the orange line with the 1 bus (based on https://api-v3.mbta.com/routes I believe the ID is `line-1`). We should verify that the code works on buses first though :)